### ISSUE TITLE
[A11Y] Language switcher does not support keyboard only navigation

### ISSUE THREAD

**Issue by israelcefrin** [pkp/immersion#125]
## Issue Description
It is not possible to switch or activate the language switcher using keybord only navigation.

## How to reproduce / Current behavior
1. Access: **https://ojs34.demo.publicknowledgeproject.org/index.php/immersion**
2. Press `TAB` key to move the focus
3. Notice that it is not possible to focus and activate the language switcher.

## What application are you using?
 OJS version 3.4

## Stack
- Device: Lapto
- OS:   MacOS
- Browser: edge, firefox, chrome

## Testing method
- Method: Manual, Keyboard

## Action
Implement  focus on language switcher to enable interaction using keyboard.

## Additional information
The component was implemented using a LINK element tag `<a > </a>` without `HREF` attribute. This element should be a `button` tag instead to provide better support to different browsers + screenreader combination.

![Image](https://github.com/user-attachments/assets/c5927a6c-98c2-4d58-a866-6789544f71f6)

**Comment by kaitlinnewson** [comment]
Thanks @israelcefrin! I've merged to `stable-3_5_0` and forward-ported to `main`. Did you want to port this to `stable-3_4_0` as well, or is it fine to just have the improvements for the 3.5 release and later?

**Comment by israelcefrin** [comment]
Hi @kaitlinnewson , I think it is fine to just have the improvements for 3.5  and main branches. Also, providing more impact on 3.5+ will state clearly that all journals that upgrade will benefit right away. Thanks for forward-ported it to `main`.
